3469/1/api.rb

```diff
@@ -0,0 +1,27 @@
+class Api
+  attr_reader :gem_information
+  def initialize(key, name_of_gem)
```

Comment:
It is bad practice to return value from initialize, you should add another public method